packages/npm/send/ios/Send.swift

```diff
@@ -18,78 +8,29 @@ private func bodyIsUTF8(contentTypeHeader: String?, utf8ContentTypes: [String])
     return utf8ContentTypes.contains(String(contentType))
 }
 
-enum SendError: Error {
-    case nonBase64RequestBody
-    
-    case nonUTF8RequestBody
-    case nonUTF8ResponseBody
-    
-    case invalidRequestBaseURL
-    case invalidRequestPathOrQueryParameters
-    case invalidResponseHeaderParameters
-    
-    var message: String {
-        switch self {
-        case .nonBase64RequestBody:
-            return "Your request headers specify a Content-Type NOT included in `utf8ContentTypes`, but your request body is not a base64-encoded string."
-        case .nonUTF8RequestBody:
-            return "Your request headers specify a Content-Type included in `utf8ContentTypes`, but your request body is not a UTF8-encoded string."
-        case .nonUTF8ResponseBody:
-            return "The response headers specify a Content-Type included in `utf8ContentTypes`, but the response body was not UTF8-encoded."
-            
-        case .invalidRequestBaseURL:
-            return "Your base URL is not valid."
-        case .invalidRequestPathOrQueryParameters:
-            return "Your path or query parameters is not valid."
-        case .invalidResponseHeaderParameters:
-            return "The response headers were not valid."
-            
-        }
-    }
-}
-
-@propertyWrapper
-struct RequiredKey<Value> {
-    var wrappedValue: Value?
-}
-
-extension RequiredKey: Decodable where Value: Decodable {
-    init(from decoder: Decoder) throws {
-        let container = try decoder.singleValueContainer()
-        wrappedValue = container.decodeNil() ? nil : try container.decode(Value.self)
+final class IgnoreRedirectsDelegate: NSObject, URLSessionTaskDelegate {
+    func urlSession(_ session: URLSession, task: URLSessionTask, willPerformHTTPRedirection response: HTTPURLResponse, newRequest request: URLRequest) async -> URLRequest? {
+        return nil
     }
 }
 
-extension RequiredKey: Encodable where Value: Encodable {
-    func encode(to encoder: Encoder) throws {
-        var container = encoder.singleValueContainer()
-        try container.encode(wrappedValue)
-    }
-}
+extension SendError: Error {}
 
-struct Request: Decodable {
-    let baseURL: String
-    let path: String
-    let queryParameters: [String: String]
-    let headerParameters: [String: String]
-    let method: String
-    @RequiredKey var body: String?
-    let utf8ContentTypes: [String]
-    
+extension Request {
     var url: Result<URL, SendError> {
         guard var urlComponents = URLComponents(string: baseURL) else {
-            return .failure(.invalidRequestBaseURL)
+            return .failure(SendError(code: .ivnalid_request_base_url, message: ""))
```

Comment:
```suggestion
            return .failure(SendError(code: .ivnalid_request_base_url, message: ""))
```

We should add back the human-readable strings...